core/static/css/base.css

```diff
@@ -20,6 +20,7 @@
     --color-pp-pale-grey: #d8d8d8;
     --color-pp-dark-grey: #55585a;
     --color-pp-almost-black: #1a1c1a;
+    --color-pp-purple: #6b4a77;
```

Comment:
As Liane mentioned, we cant use colours outside central pathogens portal's visual identity.